django-stubs/contrib/auth/base_user.pyi

```diff
@@ -41,3 +42,8 @@ class AbstractBaseUser(models.Model):
     @classmethod
     @overload
     def normalize_username(cls, username: Any) -> Any: ...
+
+# This is our "placeholder" type the mypy plugin refines to configured 'AUTH_USER_MODEL'
+# wherever it is used as a type. The most recognised example of this is (probably)
+# `HttpRequest.user`
+_UserModel: TypeAlias = AbstractBaseUser  # noqa: PYI047
```

Comment:
I know this is just a placeholder, and was chosen to not change things, but should this not be `AbstractUser` not `AbstractBaseUser`?

Looking at Django the difference between the two would be:

https://github.com/django/django/blob/39de2e97a06d0317973b280bc159ca6f89fc64e3/django/contrib/auth/models.py#L335-L341

For me I'm seeing that `is_superuser` is not defined and that's because it's checking against `AbstractBaseUser`.

Anyways, just asking since I'm assuming you know the hierarchy a little better than I do, and I will continue trying to figure out why my customized plugin might not be resolving the user model.